adafruit_ble_radio.py

```diff
@@ -142,19 +142,19 @@ def send_bytes(self, message):
         time.sleep(AD_DURATION)
         self.ble.stop_advertising()
 
-    def receive(self):
+    def receive(self, **args):
         """
         Returns a message received on the channel on which the radio is
         listening.
 
         :return: A string representation of the received message, or else None.
         """
-        msg = self.receive_full()
+        msg = self.receive_full(**args)
         if msg:
             return msg[0].decode("utf-8").replace("\x00", "")
         return None
 
-    def receive_full(self):
+    def receive_full(self, wait=1):
```

Comment:
Since this is ultimately getting passed into `ble.start_scan()` as the `timeout` argument I think it would make sense to use the same name here so `timeout` instead of `wait`. I think that would make it more clear specifically what it's going to be used for.

Reply:
Sounds good to me :)